tests/support/flattened/utility.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+// module Main
+
+exports.same = function(a) {
```

Comment:
It's quite a low-level function that I don't think is useful enough to distribute with `Test.Assert`. It's also quite unreliable as more optimisations get added later.